e2e/pages/events/uploadAdditionalApplicationsEvent.page.js

```diff
@@ -0,0 +1,72 @@
+const { I } = inject();
+const money = require('../../helpers/money_helper');
+const supportingDocumentsFragment = require('../../fragments/supportingDocuments.js');
+const supplementsFragment = require('../../fragments/supplements.js');
+module.exports = {
+  fields: {
+    uploadC2: '#temporaryC2Document_document',
+    description: '#temporaryC2Document_description',
+    usePbaPayment: {
+      yes: '#usePbaPayment-Yes',
+      no: '#usePbaPayment-No',
+    },
+    pbaNumber: '#pbaNumber',
+    clientCode: '#clientCode',
+    customerReference: '#fileReference',
+    supportingDocuments: supportingDocumentsFragment.supportingDocuments(0, 'temporaryC2Document_supportingEvidenceBundle'),
+    supplements: supplementsFragment.supplements(0, 'temporaryC2Document_supplementsBundle'),
+  },
+
+  applicationTypePrefix: '#c2ApplicationType_type-',
+  additionalApplicationTypePrefix: '#additionalApplicationType-',
+  c2AdditionalOrdersRequested:'#temporaryC2Document_c2OrdersRequested-',
```

Comment:
maybe should be moved to fields for consistency, e.g
applicationType: type => `#c2ApplicationType_type-${type}`

and then

 selectAdditionalApplicationType(type) {
    I.click(this.fields.applicationType(type));
  },